src/Compilers/CSharp/Portable/Emitter/EditAndContinue/EmitHelpers.cs

```diff
@@ -111,6 +119,37 @@ internal static class EmitHelpers
                 changedTypes: changedTypes.ToImmutableAndFree());
         }
 
+        private static bool ContainsAnonymousDelegates(
```

Comment:
Nit: Naming seems off to me. This checks for changes to delegates, not just whether they exist, right? Or does the current name mean "Contains all existing anonymous delegates"?

Reply:
The method checks the new collection contains all existing anonymous delegates. Renamed to `ContainsPreviousAnonymousDelegates` to clarify.